### PR TITLE
Combat Patch

### PR DESCRIPTION
**Pull request by NullCGT**

### CODE REVIEW

File: src/mklev.c [hunk 1]
```diff
@@ -855,7 +855,7 @@ fill_ordinary_room(struct mkroom *croom)
         mksink(croom);
     if (!rn2(60))
         mkaltar(croom);
-    if (!rn2(70))
+    if (!rn2(70) && depth(&u.uz) >= 15)
```
Comment (RojjaCebolla) [inline src/mklev.c:858]: whoa!

File: src/artifact.c [hunk 2]
```diff
@@ -73,6 +74,19 @@ hack_artifacts(void)
     artilist[ART_SUNSWORD].otyp = FIRST_SWORD + rn2(LAST_SWORD - FIRST_SWORD);
     artilist[ART_WAR_S_SWORD].otyp = FIRST_SWORD + rn2(RUNESWORD - FIRST_SWORD);
     artilist[ART_CARNWENNAN].otyp = DAGGER + rn2(STILETTO - DAGGER);
+    artilist[ART_SHAMBLESTICK].otyp = CLUB + rn2(AKLYS - CLUB);
+
+    /* Shamblestick is random, becuase what NetHack clearly needs is more randomness. */
+    artilist[ART_SHAMBLESTICK].acolor = rn2(CLR_MAX);
+    for (i = 0; i < rnd(7); i++) {
```
Comment (RojjaCebolla) [inline src/artifact.c:81]: so this gives it up to 7 "spfx", which i assume is special effects on hit? (god i hate nethack abbreviations lol)

Comment (RojjaCebolla) [reply src/artifact.c:81]: i found the info in an .h file

File: skills.txt [hunk 3]
```diff
@@ -29,6 +32,13 @@ Sneak Attack
 Stunning Fist
     Stunning fist increases the chances of stunning a monster with unarmed
     attacks, provided the user already had the ability to do so.
+Sunder
```
Comment (RojjaCebolla) [inline skills.txt:35]: imo, simplify this: disarm is for weapons, sunder is for armor & amulets. (either one _could_ be for rings i guess) i don't think we want to have to balance around 2 different ways to remove an opponent's weapon, and i don't wanna make players try to puzzle out which one is better

**Comment by RojjaCebolla** [comment]
> Wands of wishing can still be randomly generated. Gotta throw new players a bone.

i don't feel that this *does* throw new players a bone, because they play slowly.  this throws people who play _many games quickly_ a bone.  i feel like random WoWs are too game-warping, and should not happen.

**Comment by RojjaCebolla** [comment]
And in the event anyone tries to do turncount speedruns in splice, random WoWs are pretty terrible because you basically have to scum for them. Whereas, without random WoWs, there's still stuff you might scum for, but that stuff could crop up during a normal game, and you can respond to that luck by going for turncounts or conducts that you weren't originally planning on when you started.

File: doc/spl-changelog.txt [hunk 4]
```diff
@@ -228,6 +230,8 @@ Role Changes
 - Priest
   - Priests can be of any race.
 - Added a few new role/race/alignment combinations.
+- All roles automatically learn their special spell at level 12. The chance of successfully
```
Comment (RojjaCebolla) [inline doc/spl-changelog.txt]: This is really awesome! And, now we probably need to do something about the fact that some special spells are game warping (taming!) and some are useless junk (remove curse)? Whatchoo think?

Comment (NullCGT) [reply doc/spl-changelog.txt]: I absolutely 100% agree. Taming for tourists is absurdly good, for example.

**Pull request closed without merging**